src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = UserService.findByEmail(email);
+
+        if (user != null) {
+            return user.getEmail().equals(email) && user.getPassword().equals(password);
```

Comment:
we can simplify this, cause when you call findByEmail() method and it doesn’t return null, it means that user with this email already exists